frontend/src/bundles/employer-onboarding/components/onboarding-form/onboarding-form.tsx

```diff
@@ -41,12 +43,32 @@ const OnboardingForm: React.FC = () => {
 
     const dispatch = useAppDispatch();
 
+    const { currentUser } = useAppSelector((state: RootReducer) => state.auth);
+
     const onSubmit = useCallback(
         async (data: EmployerOnboardingDto): Promise<boolean> => {
-            await dispatch(actions.createEmployerDetails(data));
+            const {
+                fullName,
+                employerPosition,
+                companyName,
+                companyWebsite,
+                location,
+                description,
+            } = data;
+            await dispatch(
+                actions.createEmployerDetails({
+                    fullName,
+                    employerPosition,
+                    companyName,
+                    companyWebsite,
+                    location,
+                    description,
+                    userId: currentUser?.id,
```

Comment:
```suggestion
                    ...data,
                    userId: currentUser?.id,
```

Reply:
I made it to fiter `photo`, `companyLogo` which we don't sent to server, so here we get on those we send to server. it is the same as n the talent onboarding [here](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/blob/development/frontend/src/bundles/talent-onboarding/components/contacts-cv-step/contacts-cv-step.tsx) line 82. all my logic the same as there